Packs/QRadar/Integrations/QRadar_v3/README.md

```diff
@@ -16,6 +16,8 @@ This integration was integrated and tested with API versions 10.1-14.0 on QRadar
     | Fetch mode |  | True |
     | Number of offenses to pull per API call (max 50) |  | False |
     | Query to fetch offenses | Define a query to determine which offenses to fetch. E.g., "severity &amp;gt;= 4 AND id &amp;gt; 5 AND status=OPEN". | False |
+    | First fetch time | First fetch timestamp \(&amp;lt;number&amp;gt; &amp;lt;time unit&amp;gt;, e.g., 12 hours, 7 days\) | False |
```

Comment:
this doesnt seem to be such a clear description to me. Maybe "how long to look back while fetching incidents on the first fetch"